From 8c870727a59dfe05ff3971c053880b0cf6ee70e2 Mon Sep 17 00:00:00 2001
From: ghhccghk <2137610394@qq.com>
Date: Tue, 28 May 2024 16:06:56 +0800
Subject: [PATCH] add #include <stdlib.h>

---
 gettext-tools/src/locating-rule.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gettext-tools/src/locating-rule.c b/gettext-tools/src/locating-rule.c
index 382d162..86fab80 100644
--- a/gettext-tools/src/locating-rule.c
+++ b/gettext-tools/src/locating-rule.c
@@ -47,6 +47,7 @@
 #include <libxml/parser.h>
 #include <libxml/uri.h>
 #include "xalloc.h"
+#include <stdlib.h>
 
 #define _(str) gettext (str)
 
-- 
2.45.1

